src/config/section/compute.js

```diff
@@ -334,6 +334,12 @@ export default {
           icon: 'delete',
           label: 'label.action.destroy.instance',
           args: ['expunge', 'volumeids'],
+          mapping: {
+            volumeids: {
+              api: 'listVolumes',
+              params: (record) => { return { virtualmachineid: record.id } }
```

Comment:
Pass a type=DATADISK (check the params), that should address the issue

Reply:
Done!